samples/java/junit5-selenium/README.md

```diff
@@ -0,0 +1,28 @@
+# JUnit5 sample project
+
+## Pre-requisites for running the tests in the project
+
+- Java must be installed on your system (JDK8 or higher)
+- Maven must be installed on your system.
+- Python3 must be installed on your system.
+- Chrome browser must be installed on the local system for running the tests.
+
+## How to parse the test results to testrail without using any CI/CD tool
```

Comment:
Don't use "parse results to TestRail", always use import/submit/send in this context.
Also, **TestRail** 😄 
And there's no need to bring CI/CD into the readme. Say from your local machine if you must.

Reply:
@d-rede its done now